react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class Stopwatch extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = { isTicking: false, time: 0, switchStartClass: "start", switchPauseClass: "pause hidden"};
```

Comment:
I would store these classnames at properties rather than in state. State should hold only info that is needed in order to manage the app.